src/controllers/user.controller.js

```diff
@@ -1 +1,32 @@
-exports.status = async (req, res, next) => res.json({ message: 'User endpoint is working.' })
\ No newline at end of file
+const User = require('../models/user.model')
+const errorHandler = require('../middlewares/error-handler')
+
+exports.status = async (req, res, next) => res.json({ message: 'User endpoint is working.' })
+
+exports.add_user = async (req, res, next) => {
+  const { title, username, firstname, lastname } = req.body;
+  const newUser = {
```

Comment:
Thoughts on adding program and graduation date? @ZGao28 @callmekungfu

Reply:
yeah I think that would definitely be helpful

How should we store the graduation date tho? Like a single string or an object with more details?

Reply:
I think Grad date should just be month and year

Reply:
@davidhewwing I like that